### DISCUSSION TITLE
Serious problem with PDF files

### DISCUSSION THREAD

**Discussion by mazierovictor** [open-webui/open-webui#7733] [General]
## Installation Method
I am using Docker installation

## Environment

- **Open WebUI Version:** [e.g.v0.4.8]

- **Operating System:** Ubuntu 24.04

## Expected Behavior:

When attaching a PDF file directly to the chat, I need the file to be processed in its entirety. In other words, when the upload is complete, the system extracts all the information from the PDF, saves it in a string and waits for the user's command. This way, LLM would process the user's command according to the information extracted from the file.

Example:
I have a general model, and I need it to summarize a PDF file that is a scientific article. So, I upload the file, give it the command to summarize the file from beginning to end, and after processing, I have my summarized text.

## Actual Behavior:

What is happening is that when I upload a PDF file, openwebui understands that it is a RAG, so if I ask it to summarize a file, it does not summarize it in its entirety, hindering any attempt to interact with any type of file.

After all, if I really wanted to create a RAG, I would use the knowledge base for that.

**Bug Summary:**
The bug basically consists of a file processing confusion, that is, if I want to make, for example, a template for a ChatPDF with Openwebui, I can't, because when I upload a file, it extracts the information incompletely, or as if that file were for a RAG, because the system segments it, different from the expected behavior for the chat, where it would have to process the PDF file, extract the information and send all the extracted information to the LLM so that it is returned according to the user's request.

## Reproduction Details

**Steps to Reproduce:**
Have a registered model, without any configuration or prompt, upload a PDF file of any scientific article, in the chat, request that that article be summarized, wait for the result.

**Comment by tjbck** [comment]
You should use a whole document mode instead of retrieval mode for summarisation tasks. Hope this helps!

**Reply by mazierovictor** [reply]
@tjbck and how do I do this? can you help me?

**Reply by mazierovictor** [reply]
I managed to find it, is there any way I can set a parameter for it to always consider?

**Reply by Jeevanhm** [reply]
I'm having same issue, how did you fix it?

**Reply by auliaihendradi** [reply]
> I managed to find it, is there any way I can set a parameter for it to always consider?

Where do you find it?

**Reply by mahenning** [reply]
When uploading a document in a chat, but before sending the document, you can click on it in the chat prompt bubble. In the opened window (with the document text), there is a slider in the upper right corner with "Using Focused Review" as default. Click on the slider to change to "Using Entire Document".

**Reply by siyabongaprospersithole** [reply]
what!, nothing happens when I click the actual document

**Reply by mahenning** [reply]
![grafik](https://github.com/user-attachments/assets/190579af-e082-4a51-a3f8-0b097f74e425)
click on the document in the chat input to open the document (next image). Wait for it to upload and embedding first.

----
![grafik](https://github.com/user-attachments/assets/acf82349-08fb-42a4-8c11-1a892a592b3b)

Toggle the bubble to "Using entire Document (like in the screenshot)
